content/docs/12-enterprise-version/10-upgrade.md

```diff
@@ -13,6 +13,81 @@ fullWidth: false
 
 This page is a reference resource to help you better prepare for a Palette upgrade. Review each version's upgrade notes for more information about required actions and other important messages to be aware of. If you have questions or concerns, reach out to our support team by opening up a ticket through our [support page](http://support.spectrocloud.io/).
 
+# Palette 4.0
+
+Palette 4.0 includes the following major enhancements that require user intervention to facilitate the upgrade process. 
+
+<br />
+
+- **Enhanced security for Palette microservices** - To enhance security, all microservices within Palette now use `insecure-skip-tls-verify` set to `false`. When upgrading to Palette 4.0, you must provide a valid SSL certificate in the system console. 
+
+  <br />
+
+  If you already have an SSL certificate, key, and Certificate Authority (CA) certificate, you can use them when upgrading to Palette 4.0.0. To learn how to add a certificate to Palette, refer to [Add Certificate](/enterprise-version/upgrade#addcertificate).
+
+
+- **Upgraded on-prem Palette Kubernetes** - the Kubernetes version used for on-prem Palette is upgraded from 1.24 to 1.25. You will need to copy the new Kubernetes YAML to the Kubernetes layer in the Enterprise cluster profile. If you have customized your Kubernetes configuration, you will need to manually adjust custom values and include any additional configuration in the upgraded YAML that we provide. Refer to [Upgrade Kubernetes](/enterprise-version/upgrade#upgradekubernetes.)
+
+## Upgrade from Palette 3.x to 4.0
+
+From the Enterprise cluster system console, click the **Update version** button. Palette will be temporarily unavailable while system services update.
```

Comment:
```suggestion
From the Palette system console, click the **Update version** button. Palette will be temporarily unavailable while system services update.
```
I think we should just say Palette system console. The term enterprise cluster can be a bit confusing.